controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -143,15 +143,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                                        return isCreateProcessed(scope, kvtName, kvtConfig, createTimestamp, executor);
                                  });
                             });
-               }, e -> Exceptions.unwrap(e) instanceof RetryableException, NUM_RETRIES, executor)
```

Comment:
why is this changed?

Reply:
This is for delete and create to have consistent behavior on timeout.
After this change both throw timeout exception instead of Create returning FAILURE status on timeout.

Reply:
As discussed with Shivesh, changed both APIs to throw timeout exception when event timesout.